scipy/stats/_stats_py.py

```diff
@@ -8242,11 +8252,18 @@ def combine_pvalues(pvalues, method='fisher', weights=None):
         Zi = distributions.norm.isf(pvalues)
         statistic = np.dot(weights, Zi) / np.linalg.norm(weights)
         pval = distributions.norm.sf(statistic)
+    elif method == 'friston':
+        statistic = np.max(pvalues)
+        pval = statistic**len(pvalues)
+    elif method == 'nichols':
+        statistic = np.max(pvalues)
+        pval = statistic
```

Comment:
@fkloosterman If you can point to the location within the references I can use to verify these, I might be able to review this PR.
 
Also, it seems that we should be able to interpret these results with a statement like "Under the null hypothesis, `pval` is the probability of achieving a value of the test statistic at least as extreme as `statistic`. What are the null hypotheses according to these methods? Perhaps another way of getting at this question is: How did you know what to return for `statistic`?

Reply:
NM, looks like there is some discussion in gh-15481. Probably should come to a consensus there, and those involved can review this.